docker-compose.end-to-end-tests-Build.yml

```diff
@@ -40,6 +40,12 @@ services:
       # - AWS_ACCESS_KEY=
       # - AWS_SECRET_ACCESS_KEY=
       # URIs and Port
+      # Antivirus
+      - API_HOST=http://localhost:3030
+      - FILE_SECURITY_CHECK_SERVICE_URI=http://localhost:8081/scan/file
+      # - ENABLE_FILE_SECURITY_CHECK=true
+      # - FILE_SECURITY_SERVICE_USERNAME
```

Comment:
why not adding default values here and as we are in docker-compose, should the av service be added here into the docker compose file?